app_dart/lib/src/request_handlers/get_branches.dart

```diff
@@ -7,34 +7,20 @@ import 'dart:async';
 import 'package:meta/meta.dart';
 
 import '../datastore/cocoon_config.dart';
-import '../foundation/providers.dart';
-import '../foundation/typedefs.dart';
-import '../foundation/utils.dart';
 import '../request_handling/body.dart';
 import '../request_handling/request_handler.dart';
 
-/// Queries GitHub for the list of all available branches on
-/// [config.flutterSlug] repo, and returns list of branches
-/// that match pre-defined branch regular expressions.
 @immutable
```

Comment:
dart doc? This still gets branches from the framework repo :)

Reply:
Have removed.

Reply:
Sorry I could have been clearer. Can you add a dart doc explaining this class? There should be class level documentation for every class.

Reply:
Added.